processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -16,32 +16,37 @@ package cumulativetodeltaprocessor
 
 import (
 	"context"
-	"time"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.uber.org/zap"
 
-	awsmetrics "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor/tracking"
 )
 
 type cumulativeToDeltaProcessor struct {
-	metrics         map[string]bool
+	metrics         map[string]struct{}
```

Comment:
Can we rename this to `ConfiguredMetrics` or something like this? Also I suggestion I got from Bogdan is to write them with capital for exemplification.